.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: android
+android:
+  components:
+    # Uncomment the lines below if you want to
+    # use the latest revision of Android SDK Tools
+    - tools
+    - platform-tools
+    - tools
+
+    # The BuildTools version used by your project
+    - build-tools-27.0.2
+
+    # The SDK version used to compile your project
+    - android-27
+
+before_script: touch ~/.android/repositories.cfg
+
+notifications:
+  email: false
```

Comment:
Good call on that one

Reply:
Yup. Did not want to get my inbox flooded with emails